packages/expo-router/src/fork/getPathFromState-forks.ts

```diff
@@ -1,8 +1,8 @@
+import { matchDynamicName, matchGroupName } from '@expo/router-core';
```

Comment:
This is a bit tricky too. We'll have to be selectively what we import here and I'm not sure how to resolve some of this.

If we go against the build processes of the rest of the monorepo, we would have a hybrid ESM/CJS build. But then that'd mean we wouldn't have to create `@expo/router-core` at all.

But leaving imports here in a component path means that the client-bundle still has a path to a CJS-only package (which will be `@expo/router-core`)

`@expo/server` actually does do a hybrid build, so we could do the same here. Alternatively, we could try to share these from `@expo/server`. Unclear what's best